python/catboostserver/catboostserver/model.py

```
@@ -0,0 +1,78 @@
+        if len(model_files) > 1:
+            cbm_files = [f for f in model_files if f.endswith(".cbm")]
+            if cbm_files:
+                model_files = [cbm_files[0]]  # Use first .cbm file
+            else:
+                model_files = [model_files[0]]  # Use first available file
+
+        model_file = model_files[0]
+        self._model = cb.CatBoost()
+        try:
+            self._model.load_model(model_file)
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)